project/src/components/blocks/offers/OfferCard.jsx

```diff
@@ -31,10 +33,13 @@ export default function OfferCard(props) {
     rating,
   } = offer;
 
+  const dispatch = useDispatch();
+  const status = isFavorite ? '0' : '1';
+
   return (
     <article className={`${cardType}${cardType === CARD_TYPES.CITIES ? '__place-card' : '__card'} place-card`}
-      onMouseEnter={cardType === CARD_TYPES.FAVORITES ? null : () => onMouseEnter(id)}
-      onMouseLeave={cardType === CARD_TYPES.FAVORITES ? null : () => onMouseEnter({})}
+      onMouseEnter={cardType === CARD_TYPES.CITIES ? () => onMouseEnter(id) : null}
+      onMouseLeave={cardType === CARD_TYPES.CITIES ? () => onMouseEnter({}) : null}
```

Comment:
Можно вот так, красиво. вернется undefined, если `cardType !== CARD_TYPES.CITIES`
```suggestion
      onMouseEnter={cardType === CARD_TYPES.CITIES && () => onMouseEnter(id)}
      onMouseLeave={cardType === CARD_TYPES.CITIES && () => onMouseEnter({})}
```
Но все равно выглядит сложновато